plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -582,67 +582,45 @@ codemodernizer.explorer.show_job_history=Show job history
 codemodernizer.migration_plan.substeps.description_failed=Build failed
-codemodernizer.migration_plan.substeps.description_stopped=Job is stopped
+codemodernizer.migration_plan.substeps.description_stopped=Transformation stopped
 codemodernizer.migration_plan.substeps.description_succeed=Build succeeded
 codemodernizer.migration_summary.header.title=Transformation summary
 codemodernizer.notification.info.download.started.content=Downloading the updated code
-codemodernizer.notification.info.download.started.title=Download Started
-codemodernizer.notification.info.modernize_complete.content=Transformation job successfully finished. You can view the changes and accept changes to further test the transformed code and push it to production.
-codemodernizer.notification.info.modernize_complete.title=Transform Complete
+codemodernizer.notification.info.download.started.title=Download started
+codemodernizer.notification.info.modernize_complete.content=Transformation completed successfully! You can view the changes and accept changes to further test the transformed code and push it to production.
+codemodernizer.notification.info.modernize_complete.title=Transformation completed successfully!
 codemodernizer.notification.info.modernize_complete.view_diff=View diff
 codemodernizer.notification.info.modernize_complete.view_summary=View transformation summary
 codemodernizer.notification.info.modernize_failed.connection_failed=Unable to contact CodeWhisperer service as an exception happened:\n{0}
 codemodernizer.notification.info.modernize_failed.description=The job failed with error: {0}{1}
 codemodernizer.notification.info.modernize_failed.failure_is_retryable=, please try again in a bit.
-codemodernizer.notification.info.modernize_failed.title=Transform failed
+codemodernizer.notification.info.modernize_failed.title=Transformation failed
 codemodernizer.notification.info.modernize_failed.unknown_failure_reason=unknown reason
 codemodernizer.notification.info.modernize_ongoing.view_status=View status
-codemodernizer.notification.info.modernize_partial_complete.content=Transformation job partially successful! The new code is not fully Java {0} compatible yet. You will need to fix changes yourself. You can view the changes by clicking view diff below.
-codemodernizer.notification.info.modernize_partial_complete.title=Transformation partially successful!
-codemodernizer.notification.info.submitted_project.content=Your project has been submitted for transformation. The code transformation process may take 10-30 mins depending on the size of your project.
-codemodernizer.notification.info.submitted_project.title=Submitted Project
+codemodernizer.notification.info.modernize_partial_complete.content=Transformation completed, but with build errors. The new code is not fully Java {0} compatible yet. You will need to fix changes yourself. You can view the changes by clicking View Diff below.
```

Comment:
Already reviewed with PM, he wants this change to say "...but with build errors" instead of "partially completed"